src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -170,4 +171,22 @@ public void setSortingMethod(SortingMethod sortingMethod) {
     public SortingMethod getSortingMethod() {
         return this.sortingMethod;
     }
+
+    /**
+     * Adds properties files which house classifications in key value pairings. When these properties files get
+     * processed these classifications get displayed on the main page of the report as metadata in the order in which
+     * they appear within the file.
+     *
+     */
+    public void addPropertiesFiles(List<String> propertyFiles) {
```

Comment:
`addClassificationFiles` - there is clear javadoc and example in .README which tell we expect properties, not eg xml files